public/locales/en/sensors.json

```diff
@@ -102,5 +102,49 @@
         "confirmPurpleAirId": "Type the PurpleAir ID of the sensor to confirm deletion",
         "whichSensorToDelete": "Which sensor would you like to delete?",
 		"error": "The sensor was not able to be deleted. Please try again later."
+    },
+    "sensorErrors": {
+        "heading": "Recent Sensor Errors",
+        "explanation": "These errors occurred the most recent time we attempted to get a reading for this sensor from PurpleAir.",
+        "readingNotReceived": {
+            "heading": "Reading Not Received",
+            "explanation": "No new reading was received from PurpleAir the last time we attempted to get a reading."
+        },
+        "noHumidity": {
+            "heading": "No Humidity Reading Received",
+            "explanation": "No humidity reading was received for this sensor. The humidity reading is necessary to apply the EPA correction to the PurpleAir readings. This may indicate that the humidity sensor is malfunctioning."
+        },
+        "incompleteReading": {
+            "heading": "Incomplete Sensor Reading",
+            "explanation": "The last reading we received from PurpleAir was missing at least one field, so we cannot use the reading. This may indicate sensor malfunction."
+        },
+        "channelsDiverged": {
+            "heading": "Channels Diverged",
+            "explanation": "The readings from Channel A and Channel B of the sensor have gotten too far apart to be able to use the reading in the AQI calculation. This may indicate that one or both of the channels has malfunctioned."
+        },
+        "channelADowngraded": {
+            "heading": "Channel A Downgraded",
+            "explanation": "Channel A has been downgraded by PurpleAir, likely indicating sensor malfunction."
+        },
+        "channelBDowngraded": {
+            "heading": "Channel B Downgraded",
+            "explanation": "Channel B has been downgraded by PurpleAir, likely indicating sensor malfunction."
+        }
+    },
+    "aqiErrors": {
+        "heading": "Recent AQI Calculation Errors",
+        "explanation": "These errors are possible reasons that a sensor does not currently have a valid AQI.",
+        "tooHigh": {
+            "heading": "AQI Too High",
+            "explanation": "The calculated AQI was higher than the possible AQI range, which likely means that the sensor is malfunctioning."
```

Comment:
nit: perhaps change to "the calculated AQI was over 1000, which is not possible" or something like that ("the calculated AQI was infinite.."). I'm just wondering if they get this error and try to communicate with PurpleAir they might run into a communication issue w/ the fact that the AQI doesn't really have a max (like 500 is not actually the max even though they say it is). I'm picturing someone telling PA it's higher than possible (without knowing the number) and PA responding that there is no max val.

Reply:
I don't feel strongly about this though, so feel free to ignore.

Reply:
Maybe I'll add that the sensor is reporting impossibly high PM2.5 readings